optax/_src/alias_test.py

```diff
@@ -120,7 +122,7 @@ def test_optimization(self, opt_name, opt_kwargs, target, dtype):
           f'{opt_name} does not support complex parameters.'
       )
 
-    opt = getattr(alias, opt_name)(**opt_kwargs)
+    opt = getattr(contrib if opt_name == 'cocob' else alias, opt_name)(**opt_kwargs)
```

Comment:
I'm using the tests in `alias`, but let me know if you want me to do something different.

Reply:
Could you have a cocob_test.py file in contrib/ instead?
It's fine if you duplicate much of this code from this file into that file?
(I will then later extract the repeated code in some test utils)